### DISCUSSION TITLE
MeshCentral config file

### DISCUSSION THREAD

**Discussion by mikesellt** [tteck/Proxmox#2120] [Help Needed] [answered]
Hello, first of all thank you for the scripts!  I've used probably 10 of them, and they're working great for the most part.  I do have an issue trying to run MeshCentral.  The script seems to run fine, everything updates, and I can get to the web gui and create a login.  I'm trying to change the mode to WAN (among other things) via the config file, but I can't seem to find the config file.  The docs say it should be located in ~/meshcentral-data/config.json, but that folder doesn't exist.  I do see the config file section of the web gui, but I can't edit it.   That may be by design though, as the documentation seems to suggest changes should be made from nano, vi, etc...

Am I missing a step in the installation or script?  I ran with default settings.  

Thanks,

Tim

**Comment by tteck** [comment, marked as answer]
```
nano /opt/meshcentral/meshcentral-data/config.json
```

**Reply by mikesellt** [reply]
Thank you!  And thanks so much for answering quickly.
